### PR TITLE
WayScript Quickstart and Install Plan

### PR DESCRIPTION
**Pull request by zstix**
## Description

Adds the initial configuration for a WayScript quickstart and install plan. The primary install method (nerdlet) hasn't been created yet, so this will be in draft mode for now.

## Remaining Work

- [ ] Verify copy is correct
- [x] Create install nerdlet (NR-1004)
- [x] Update install plan to reference nerdlet ID

## Related Issue(s)

- NR-1362
- NR-1363

### CODE REVIEW

File: quickstarts/wayscript/config.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+summary: |
+  WayScript is an Internal Developer Platform (IDP) that enables scalable, single click development environments. Connect your environments to New Relic in minutes.
+
+description: |+
+  ## Scalable, single click development environments with WayScript
+  Spin up customizable development environments which integrate Docker, Kubernetes, New Relic, and all your third-party dev tooling in a single click.
+
+  ### What is [WayScript](https://t.sidekickopen14.com/s3t/c/5/f18dQhb0S7kF8cFB-bW2kCdVL59hl3kW7_k2841CX6NGW36PLj07t5S8DW1Dh2LL96WRJDf197v5Y04?te=W3R5hFj4cm2zwW4myBCV3FcKb-w4fdHZS48S2&si=8000000020074708&pi=d3941e79-14e2-4cd1-c97d-45af92f6d865)
+  WayScript is an **Internal Developer Platform (IDP)** that enables scalable, single click development environments. The platform empowers early stage, high growth engineering teams with scalable, modern infrastructure so they can focus on product instead of battling Ops.
```
Comment (louisleungk) [inline quickstarts/wayscript/config.yml:13]: Can we keep the hyperlink within the body text and outside of the header?

File: install/wayscript/install.yml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  WayScript is an Internal Developer Platform (IDP) that enables scalable, single click development environments. Connect your environments to New Relic in minutes.
+
+target:
+  type: integration
+  destination: cloud
+
+install:
+  mode: nerdlet
+  destination:
+    nerdletId: todo-add-nerdlet-id
```
Comment (aswanson-nr) [inline install/wayscript/install.yml:17]: Just tossing a comment here so we don't forget to update this before merging.

Comment (aswanson-nr) [reply install/wayscript/install.yml:17]:
```suggestion
    nerdletId: external-install.install 
```

File: quickstarts/wayscript/config.yml [hunk 3]
```diff
@@ -0,0 +1,58 @@
+name: wayscript
```
Comment (josephgregoryii) [inline quickstarts/wayscript/config.yml:1]:
```suggestion
slug: wayscript
```

File: install/wayscript/install.yml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+id: wayscript
+
+name: WayScript
```
Comment (josephgregoryii) [inline install/wayscript/install.yml:3]:
```suggestion
displayName: WayScript
```

File: install/wayscript/install.yml [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+install:
+  mode: nerdlet
+  destination:
+    nerdletId: external-install.install 
+
+fallback:
+  mode: link
+  destination:
+    url: https://wsxdocs.wayscript.com/quickstart
```
Comment (josephgregoryii) [inline install/wayscript/install.yml:22]: https://github.com/newrelic/newrelic-quickstarts/blob/main/data-sources/argocd/config.yml#L8-L17

**Review by josephgregoryii — CHANGES_REQUESTED** [review]

File: install/wayscript/install.yml (hunk 4 above)
Comment (josephgregoryii) [reply install/wayscript/install.yml:3]: ignore this

File: install/wayscript/install.yml (hunk 5 above)
Comment (josephgregoryii) [reply install/wayscript/install.yml:22]: ignore this

**Pull request merged**